lib/internal/Magento/Framework/HTTP/PhpEnvironment/RemoteAddress.php

```diff
@@ -120,7 +125,7 @@ function (string $ip) {
     public function getRemoteAddress(bool $ipToLong = false)
     {
         if ($this->remoteAddress !== null) {
-            return $ipToLong ? ip2long($this->remoteAddress) : $this->remoteAddress;
+            return $ipToLong && $this->remoteAddress ? ip2long($this->remoteAddress) : $this->remoteAddress;
```

Comment:
Can we revert this? The goal on this one is literally just to mark classes/interfaces as API and not make any code change. If this is required because of `strict_types=1`, lets remove it and create a separate PR for it.